bin/gimme-aws-creds

```diff
@@ -141,48 +261,73 @@ class GimmeAWSCreds(object):
 
         # get the config dict
         conf_dict = config.get_config_dict()
-        config.get_user_creds()
 
-        if conf_dict['idp_entry_url'] in [None, '']:
-            print('No IDP entry URL in configuration.  Try running --config again.')
+        if conf_dict['okta_org_url'] in [None, '']:
+            print('No Okta organization URL in configuration.  Try running --config again.')
+            sys.exit(1)
 
-        idp_entry_url = conf_dict['idp_entry_url'] + '/api/v1'
+        okta = OktaClient(conf_dict['okta_org_url'], config.verify_ssl_certs)
+        if config.username is not None:
+            okta.set_username(config.username)
 
-        # create otka client
-        api_key = self._get_okta_api_key(
-            config.username,
-            config.password,
-            idp_entry_url,
-            cerberus_url=conf_dict['cerberus_url']
-        )
+        # Call the Okta APIs and proces data locally
+        if conf_dict['gimme_creds_server'] == 'internal':
```

Comment:
`.get()` here as well, otherwise will bomb out before it gets to the else statement.